corehq/motech/repeaters/utils.py

```diff
@@ -58,17 +58,25 @@ def get_common_attrs_diff(cls, couch, sql):
             cls.diff_value('repeater_id', couch.get('_id'), sql.repeater_id)
         )
 
+        # Ignore nullish value in format
+        if couch.get('format') and sql.format:
```

Comment:
Should this use `or` instead of `and`? Or are they considered the same if either of them is null/empty?
```suggestion
        if couch.get('format') or sql.format:
```

Reply:
I think it would be `and` because on the couch we can either have `''` or `None` and in SQL we will have only `None` so we want to consider `''` and `None` as the same.
But if one is nullish and another one has some value in it, it would mean that the properties are not equal. So we want to compare them when one of them is not nullish. So `and` would be better suited for this case.

Reply:
> we want to compare them when one of them is not nullish

I think this is the problem. With `and`, if either one is nullish it will skip the diff. For example
```py
>>> couch = {"format": None}
>>> class sql:
...     format = "something"
... 

>>> bool(couch.get('format') and sql.format)
False
```
Do you agree that is a problem?

Maybe it should be
```py
>>> (couch.get('format') or sql.format) and (couch.get('format') != sql.format)
True
```

```suggestion
        if (
            (couch.get('format') or sql.format)
            and (couch.get('format') != sql.format)
        ):
```
Although I'm not sure the `!=` comparison is necessary. That might be redundant with what is done in `cls.diff_value(...)`?